### ISSUE TITLE
Update the error state for "username already exists"

### ISSUE THREAD

**Issue by clicksave** [cartridge-gg/controller#671]
Lets make it easier to move from log in to sign up when a user inputs a invalid/taken username

<img width="1168" alt="image" src="https://github.com/user-attachments/assets/3530c69f-4b53-4032-ae37-3c214134e04e">

Doing a few things here 

- [ ] removes the alert icon
- [ ] changes the copy
- [ ] adds an inline 'Log in button' 

<img width="858" alt="image" src="https://github.com/user-attachments/assets/da520c70-bcb0-48be-a679-01155c7f0509">

[Figma](https://www.figma.com/design/OSbzNkyR9UYV4Erz9oF9JJ/Controller?node-id=3794-9077&t=BNkHGn3BbXI9OmBG-1)

---

Lets use a similar pattern on sign up as well 

<img width="1120" alt="image" src="https://github.com/user-attachments/assets/46f32fce-af4b-4f6f-aaa5-ab27a020cec7">

[Figma](https://www.figma.com/design/OSbzNkyR9UYV4Erz9oF9JJ/Controller?node-id=5987-50579&t=BNkHGn3BbXI9OmBG-1)

**Comment by bitfalt** [comment]
hii! I'd love to work on this. Just to get the task clear, I would need to update the error state and make it like the Figma designs which would be to fulfill the tasks you mentioned:

- [ ] Remove the alert icon
- [ ] Changes the copy (I would like to know what you mean exactly by this)
- [ ] Add an inline 'Log in' button.

**Comment by clicksave** [comment]
gm @bitfalt you can have this ticket if you'd like but I just wanted to confirm. The controller isn't an open source project so theres no bounty for this work. lmk if you still want it

**Comment by bitfalt** [comment]
okay okay, no worries! I'll still be happy to help :D

**Comment by clicksave** [comment]
Awesome! ok, to answer your question, I'm referring to the contents of the error message 

- [ ] The copy for the error state on the Sign Up page currently reads "Controller with username not found" it should instead read "Username not found" 
- [ ] The copy for the error state on the Log In page currently reads "Account already exists" it should instead read "Sorry, [username] already exists." 

You can test it yourself here if you'd like 
https://flippyflop.gg/

**Comment by clicksave** [comment]
> hii! I'd love to work on this. Just to get the task clear, I would need to update the error state and make it like the Figma designs which would be to fulfill the tasks you mentioned:
> 
> * [ ]  Remove the alert icon
> * [ ]  Changes the copy (I would like to know what you mean exactly by this)
> * [ ]  Add an inline 'Log in' button.

This is correct by the way! and you'd do it for both the log in and sign up pages

**Comment by bitfalt** [comment]
Alright, sounds good! I'll start working on this issue and do a draft PR, so you can check my progress.
